core/primitives/src/views.rs

```diff
@@ -251,6 +271,7 @@ pub struct QueryError {
 pub struct AccessKeyInfoView {
     pub public_key: PublicKey,
     pub access_key: AccessKeyView,
+    pub proof: Vec<Arc<[u8]>>,
```

Comment:
this proof is never actually used and can be removed (the real proof is inside the AccessKeyView struct)

Reply:
Great callout